xapian-applications/omega/query.cc

```diff
@@ -688,7 +702,7 @@ p_nottag(unsigned int c)
 // FIXME: shares algorithm with indextext.cc!
 static string
 html_highlight(const string &s, const string &list,
-	       const string &bra, const string &ket)
+		const string &bra, const string &ket)
```

Comment:
This line is now mis-indented (it's not clear in the diff, but `html_highlight(` is 15 characters, and you've changed the line after it to be indented with 2 tab characters, which is a width of 16 characters.